docs/Deploying/Configuration.md

```diff
@@ -2553,3 +2553,144 @@ stored in your database.
   packaging:
     global_enroll_secret: "xyz"
   ```
+
+##### packaging.s3_bucket
+
+Name of the S3 bucket to use to store pre-built Orbit installers.
+
+- Default value: ""
+- Environment variable: `FLEET_PACKAGING_S3_BUCKET`
+- Config file format:
+
+  ```
+  packaging:
+    s3:
+      bucket: some-bucket
+  ```
+
+##### s3_access_key_id
+
+AWS access key ID to use for S3 authentication.
+
+If `s3_access_key_id` and `s3_secret_access_key` are omitted, Fleet will try to use
+[the default credential provider chain](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+
+The IAM identity used in this context must be allowed to perform the following actions on the bucket: `s3:GetObject`, `s3:ListBucket`.
```

Comment:
most of the contents here are copy-pasta from carves, but things like this have been edited accordingly.